tests/scripts/test_treelite_python.py

```diff
@@ -16,13 +17,21 @@ def test_treelist_inferencing_script(temporary_dir, regression_inference_sample,
     predictions_dir = os.path.join(temporary_dir, "predictions")
 
     script_args = [
-        "score.py",
-        "--data", regression_inference_sample,
+        "compile_treelite.py",
         "--model", regression_model_sample,
-        "--output", predictions_dir,
```

Comment:
here your compile script should generate a model in an --output that is located in a temporary dir (ex: rename predictions_dir above this with compiled_model_dir), then this same directory is fed into the next module as a --model argument.

Reply:
now that you've added --so_path in compile and in score, could you add it to the unit tests as well to make sure we're consuming the model generated from compile?